src/main/java/com/mabl/integration/jenkins/test/output/TestSuite.java

```diff
@@ -1,12 +1,15 @@
 package com.mabl.integration.jenkins.test.output;
 
+import org.jaxen.pantry.Test;
```

Comment:
This is a strange import. IntelliJ pulled something in or did you expect it to come from that package?

Reply:
D'oh.